server.js

```diff
@@ -6,6 +6,11 @@ var formidable = require('formidable'),
 
 var DIRECT_UPLOAD_LIMIT = 32; // bytes
 
+// convert from UTF-8 to ISO-8859-1
+var LATIN1_SYMBOLS = '¥§©ÆÖÑøøø¼';

```

Comment:
This string is copied verbatim in _another repository_, which is very difficult to maintain. Is there a way to make the test independent of the string that gets passed?